src/retrace/retrace.py

```diff
@@ -2122,8 +2124,25 @@ def strip_extra_pages(self) -> None:
             log_warn("makedumpfile exited with %d" % child.returncode)
             if newvmcore.is_file():
                 newvmcore.unlink()
+            return 
+
+        #Calculate the % of storage space saved with the new makedumpfile core created. 
+        elif ((oldsize - newvmcore.stat().st_size) / int(oldsize) * 100 < CONFIG["VmcoreDumpSavePercent"]):
```

Comment:
The outer parentheses can be removed. Also, change the `elif` to `if`, please, as the previous `if` ends with a `return`. Also, is `int()` necessary in here?

Reply:
I found that I could not remove the outer parentheses here unless I wanted to move it to one line. As the rest of the code is all aligned I went ahead and adjusted this so it is on two lines and kept the outer parentheses. Hope that is ok and happy to change it to one line if you want.